### DISCUSSION TITLE
What is the smallest integer that, when ✖️ by 3, gives only 4s?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1018] [Galaxy Trivia] [answered]
**Bonus:** What is the largest? Consider the integer 2^32, math operations don't overflow.

**Comment by peterbaccaro** [comment]
In order to calculate the smallest integer that, when multiplied by 3, gives only 4s you can work backwards and look for a number consisting of 4's that is divisible by 3.

4 ÷ 3 = 1.3333 (not an integer)
44 ÷ 3 = 14.6666 (not an integer)
444 ÷ 3 = **148** 

Therefore the smallest integer that, when multiplied by 3, gives only 4s is **148**.

The largest integer that, when multiplied by 3, gives only 4s is **148148148**.

444444444 ÷ 3 = **148148148**.

**Comment by thuongtruong1009** [comment, marked as answer]
To find the smallest integer that, when multiplied by 3, gives only 4s, we can start by considering the number of digits in the resulting product. Since we are looking for a number consisting only of 4s, the product must have a string of 4s and no other digits. 

Let's examine the pattern formed by multiplying increasing integers by 3:

1 × 3 = 3
2 × 3 = 6
3 × 3 = 9
4 × 3 = 12
5 × 3 = 15
...

We can observe that the pattern repeats after every multiple of 3. So, for a number to consist only of 4s when multiplied by 3, the number of 4s in the product must be a multiple of 3. We can determine the smallest possible number of 4s by finding the smallest multiple of 3 that is greater than or equal to the number of digits in 2^32.

The number of digits in 2^32 is 10 because 2^32 is approximately 4,294,967,296, which has 10 digits.

To find the smallest multiple of 3 greater than or equal to 10, we can divide 10 by 3 and round up to the nearest whole number:

10 / 3 = 3.333...

Rounding up, we get 4. So, the smallest number of 4s needed in the product is 4.

To obtain a product with four 4s, we need to find the smallest integer such that n × 3 = 44,444. Dividing 44,444 by 3 gives us 14,814.666... (repeating), which means we need a fractional part of 0.666... to round up to the nearest whole number. 

Since the fractional part is repeating, we cannot have a whole number solution. Therefore, there is no integer that, when multiplied by 3, gives a product consisting only of 4s.

As for the largest integer, if there is no solution for the smallest integer, there will also be no solution for the largest integer.

**Comment by virajchandra51** [comment]
In order to calculate the smallest integer that, when multiplied by 3, gives only 4s you can work backwards and look for a number consisting of 4's that is divisible by 3.

4 ÷ 3 = 1.3333 (not an integer)
44 ÷ 3 = 14.6666 (not an integer)
444 ÷ 3 = 148

Therefore the smallest integer that, when multiplied by 3, gives only 4s is 148.

The largest integer that, when multiplied by 3, gives only 4s is 148148148.

444444444 ÷ 3 = 148148148.
